conda_build_config.yaml

```diff
@@ -0,0 +1,30 @@
+c_compiler:         # [win]
+  - vs2008          # [win]
+  - vs2015          # [win]
+cxx_compiler:       # [win]
+  - vs2008          # [win]
+  - vs2015          # [win]
+hdf5:
```

Comment:
Meh?  I don't really know.  I guess if we're doing a total rebuild, then no, not really.  We should probably only have the current major.minor version until we need to support more than one for whatever reason.